scripts/py_to_ino_RGB_LED_test.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Importing Libraries
+# For Arduino Communication:
+# https://create.arduino.cc/projecthub/ansh2919/serial-communication-between-python-and-arduino-e7cce0
+
+# import serial
+# Refer to pyserial docs: https://pyserial.readthedocs.io/en/latest/pyserial.html
+# python3 -m pip install pyserial
+# import time
+import numpy as np
+import rospy
+
+
+# ROS Data Types
+
+# Custom Script
+import image_inputs as input_image
+import light_painting.msg #custom message type
+import RGBState
+
+
+# arduino = serial.Serial(port='/dev/ttyACM0',baudrate=9600,timeout=0.1)
+# Code from: https://www.electronicshub.org/controlling-arduino-led-python/

```

Comment:
Remove all `serial` commands from script since we're using the roslaunch & rosserial package to communicate with Arduino.